src/storage/query/QueryBaseProcessor.inl

```diff
@@ -101,13 +102,13 @@ cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::checkAndBuildContexts(const REQ&
                 break;
             }
             case cpp2::PropOwner::EDGE: {
-                EdgeType edgeType = col.id.get_edge_type();
-                auto edgeName = this->schemaMan_->toEdgeName(spaceId_, std::abs(edgeType));
+                EdgeType edgeType = std::abs(col.id.get_edge_type());
```

Comment:
Return positive edgeType for reverse edge?  I think it is not correct. 
Currently,  we take reverse edge as a special edge type.  So when you want to traverse the reverse edge,  it should return the negative edge type.

Reply:
But i notice that edgeSchemaResp_ only contains abs(edgeType)

Reply:
The schema always to be same for both direction. But data is different.

Reply:
Don't do that.  For reverse edge, we should return negative edgeType.  So QE could distinguish which one is reversely.